scss/samples/_colors.scss

```diff
@@ -207,7 +207,7 @@ $demo-noncolors: (
 ///     meta.get-function('color', $module: 'tools') // function
 ///   );
 ///
-///   @each $key, $value in $herman {
+///   @each $key, $value in utilities.$herman {
```

Comment:
@mirisuzanne I'm not sure I caught all of the places where this should be changed, but this is what was breaking the build.